_includes/checkoutv3-purchase.md

```diff
@@ -46,7 +46,8 @@ Content-Type: application/json
             "nationalIdentifier": {
                 "socialSecurityNumber": "{{ page.consumer_ssn_se }}",
                 "countryCode": "SE"
-            }
+            },
+            "restrictedToSocialSecurityNumber": true,
```

Comment:
This .md is not really in use. We use the implementation specific payment order .mds, but since this feature is somewhat uncommon, I think we should leave it out from the "basic" examples and just have it present in the specific feature example.